doc/compressed_transactions.md

```diff
@@ -0,0 +1,178 @@
+
+### 3.6.1 Compressed Txid/Signature Data
+
+| Name              | Width     | Description |
+|-------------------|-----------|-------------|
+| Hash Type         | 0-1 Bytes | An Optional Byte containing the Hash Type if it was non-standard.|
+| Txid Block Index  | 1-5 Bytes | A Bitstream VarInt containing the flattened index from the Txid Blockheight for the Vout. |
+| Signature         | 64 Bytes  | Contains the 64 byte signature. |
+
+### 3.6.2 Uncompressed Txid/Signature Data
+
+| Name      | Width     | Description |
+|-----------|-----------|-------------|
+| Txid      | 32 Bytes  | Contains the 32 byte Txid. |
+| Vout      | 1-5 Bytes | A VarInt Containing the Vout of the Txid. |
+| Signature | 2+ Bytes  | A VLP-Bytestream containing the signature. |
+
+### 3.7 Output Data
+
+| Name          | Width     | Description |
+|---------------|-----------|-------------|
+| Output Script | 2+ Bytes  | A VLP-Bytestream containing the output script. |
+| Amount        | 1-9 Bytes | A Bitstream VarInt containing the output amount. |
+
+## 4 Ideal Transaction
+
+The target transaction for the most optimal compression was chosen
+based off the most common transactions that are likely to be used
+for purposes that requires the best compression.
+
+| Field           | Requirements                      | Possible Savings                  |
+|-----------------|-----------------------------------|-----------------------------------|
+| Version         | Less than four                    | 30 Bits                           |
+| Input Count     | Less then four                    | 30 Bits                           |
+| Output Count    | Less then four                    | 30 Bits                           |
+| LockTime        | 0                                 | 30 Bits                           |
+| Input Sequence  | 0x00, 0xFFFFFFFE, or 0xFFFFFFFF   | 62 Bits For Each Input            |
+| Input Txid      | Compressed Outpoint               | 23-31 Bytes For Each Input        |
+| Input Vout      | Compressed Outpoint               | (-1)-3 Bytes For Each Input       |
+| Input Signature | Non-custom Script Signing         | 40-72 Bytes For Each Legacy Input |
+| Input Hash Type | 0x00 for Taproot, 0x01 for Legacy | 7 Bits For Each Input             |
+| Output Script   | Non-custom Scripts                | 2-5 Bytes For Each Output         |
+| Output Amount   | No Restrictions                   | (-1)-7 Bytes For Each Output      |
+
+## 5 Test Vectors
+
+| Transaction              | Before Compression | Possible Savings         | After Compression |
+|--------------------------|--------------------|--------------------------|-------------------|
+| 2-(input/output) Taproot | 312 Bytes          | 78-124 Bytes and 2 Bits  | 190-226 Bytes     |
+| 2-(input/output) Legacy  | 394 Bytes          | 118-196 Bytes and 2 Bits | 176-244 Bytes     |
+
+Taproot Transaction
```

Comment:
In 6a117eac0af155610bb7bcfa30b6b878ca9628f4:

Should change these headings to `Taproot (uncompressed)` and `Taproot (compressed)`.

Reply:
Fixed, Thanks!